.github/actions/pr_notifier/pr_notifier.py

```diff
@@ -126,17 +126,12 @@ def needs_api_review(labels, repo, pr_info):
     # repokitten tags each commit as pending unless there has been an API LGTM
     # since the latest API changes. If this PR is tagged pendding it needs an
     # API review, otherwise it's set.
-    headers, data = repo._requester.requestJsonAndCheck(
-        "GET",
-        ("https://api.github.com/repos/envoyproxy/envoy/statuses/" + pr_info.head.sha),
-    )
-    if (data and data[0]["state"] == 'pending'):
-        return True
-    return False
+    return repo.get_commit(pr_info.head.sha).get_statuses()[0].state == "pending"
```

Comment:
i think this probably increases the api call count - i dont see a way to directly get status for a commit sha using the api

i think its probably still preferable to not use the private method to get this